tests/integration/tier0/inhibit-if-oracle-system-uses-not-standard-kernel/main.fmf

```diff
@@ -1,6 +1,21 @@
-summary: inhibit-if-oracle-system-uses-not-standard-kernel
+summary: Oracle Linux unsupported kernel
+description: |
+    Install unsupported Unbreakable Enterprise Kernel (uek) on an Oracle Linux machine
```

Comment:
```suggestion
    Install unsupported Unbreakable Enterprise Kernel (UEK) on an Oracle Linux machine
```

Reply:
> Just minor thing with the name Convert2RHEL. That is what we call it for the product offering. Here upstream it is convert2rhel though, so just making it consistent. Also changed some OAMG links to their RHELC counterpart after we moved to RHELC
> 
> Otherwise the code after skimming seemed good. Just feel free to merge at your own accord

Thanks @SpyTec, noted!